City-of-Helsinki/drupal-module-helfi-tpr
# UHF-1048: Accessibility sentences

To test this:

- `drush updb`
- Migrate few tpr units (`MIGRATE_LIMIT=10 drush mim tpr_unit`)
- Run tpr unit extras migrate (`drush mim tpr_unit_extra`)

Make sure `Accessibility sentences`  field is visible (Structure -> Tpr unit -> display) and that you can see migrated accessibility sentence strings somewhere in Unit's entity view.
